### ISSUE TITLE
Email: Comment: WCAG 3.0 accessibility guidelines: Accessing a "virtual" meeting using a teleconferencing application such as Zoom - receipt of streamed text content

### ISSUE THREAD

**Issue by jspellman** [w3c/wcag3#279]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Mar/0003.html): 

From: Rod Macdonald (rjmacdonald@hawaiiantel.net)

Re: W3C Accessibility Guidelines (WCAG) 3.0 - W3C First Public Working Draft 21 January 2021

Issue: Accessing a "virtual" meeting using a teleconferencing application such as Zoom - receipt of streamed text content

Background: For purposes of this discussion, individuals who are Deaf-Blind can be grouped into four sub-groups:

(1) The Deaf-Blind individual retains sufficient residual hearing, with amplification and/or other enhancements, to access web content in the of a hearing person.

(2) The Deaf-Blind individual retains sufficient residual vision, with screen magnification and/or other enhancements, to access web content in the manner of a sighted person.

(3) The Deaf-Blind individual cannot access web content via speech or hearing, but can do so using braille.

(4) The Deaf-Blind individual cannot access web content using vision, hearing or braille, and thus cannot access web content at all. (There may be extremely rare cases when the use of unusual technology may circumvent this.)

This discussion refers exclusively to Deaf-Blind individuals in the third group - braille users.

Problem: Spoken language is typically at about 200 words per minute. Adding additional information, such as identifying the speaker and adding punctuation, increases the volume of text being transmitted. The text is never enhanced for braille display.

The average adult braille reader reads at about 15 words per minute. This speed assumes error-free content, somewhat shortened into "contracted" (or "Grade 2) literary braille correctly formatted on paper.

The speed at which streamed text is rendered makes it impossible for 99% of the braille-reading population to read it. The process is made even more impossible by the fact that the braille reader is typically using a braille display that can only hold 40 characters at a time; after the 40-character limit has been reached the display jumps to a new line to continue with incoming text. No existing braille device I am aware of will allow the user to control this streaming process. In reading a document one can "pan" text forward, one 20-character display at a time; but when the input is a continuous stream the braille device will always follow the dynamic cursor - one cannot go back and read prior text while the input stream is active.

For all practical purposes a braille reader can NOT access streamed text during a Zoom meeting. There is no known solution to this accessibility issue.

The only known workaround would be for the meeting to employ a designated CAN (Computer Assisted Notetaker) interpreter to provide real-time communication for the Deaf-Blind consumer. However, a CAN interpreter typically types at about 50 words per minute and must therefore condense the communication, especially when speaker-identification is required, and the physical separation of the meeting and Deaf-Blind consumer makes for additional logistical issues - is the interpreter placed at the meeting site or at the consumer's end? How can the consumer communicate with the interpreter if they are separated? Can the funding source pay for an interpreter located off-site? In any case this solution does not address the problem with the teleconferencing application itself.

(Additional note: Youtube transcripts usually do not identify the speaker.)

**Comment by mraccess77** [comment]
Other sources, such as a national Federation of the blind indicate that average adult braille reading rates are 70 to 100 words per minute.
